pkg/cmd/load_plugins.go

```diff
@@ -113,7 +114,7 @@ func loadPlugins(baseCmd *cobra.Command, out io.Writer, pluginType string) {
 				}
 
 				// Invoke plugin
-				err = plug.Invoke(os.Stdin, out, os.Stderr, env)
+				_, err = plug.Invoke(context.Background(), &plugin.Input{Stdin: os.Stdin, Stdout: out, Stderr: os.Stderr})
```

Comment:
why did you opt to add stdout and stderr to `Input` rather than `Output` struct? My thought was it would be:
```go
type Input struct {
        // the plugin type-specific  versioned message struct
	Message any
	Stdin io.Reader
}

type Output struct {
        Message any
	Stdout, Stderr io.Writer
}
```

and to be clear, i was thinking Invoke() would add Output as a param too, and error would be the only return value.

Reply:
> and to be clear, i was thinking Invoke() would add Output as a param too, and error would be the only return value.

Loosely, I took inspiration from https://pkg.go.dev/net/http#RoundTripper (context should be a function parameter as well, it isn't on round tripper for legacy reasons)